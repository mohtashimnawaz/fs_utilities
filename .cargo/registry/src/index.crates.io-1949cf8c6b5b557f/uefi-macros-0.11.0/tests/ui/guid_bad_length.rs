use uefi::Guid;
use uefi_macros::guid;

// Fail because the length is wrong.
const TooShort: Guid = guid!("aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaa");

fn main() {}
