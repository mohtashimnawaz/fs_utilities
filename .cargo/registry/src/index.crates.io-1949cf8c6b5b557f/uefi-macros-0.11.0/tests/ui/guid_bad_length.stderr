error: "aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaa" is not a canonical GUID string (expected 36 bytes, found 35)
 --> tests/ui/guid_bad_length.rs:5:30
  |
5 | const TooShort: Guid = guid!("aaaaaaaa-aaaa-aaaa-aaaa-aaaaaaaaaaa");
  |                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
